app-modules/authorization/tests/Feature/Filament/Resources/User/ViewUserTest.php

```diff
@@ -131,3 +131,190 @@
     expect($second->isImpersonated())->toBeTrue();
     expect(auth()->id())->toBe($second->id);
 });
+
+it('does not display the mfa_status Action for an external User', function () {
+    $user = User::factory()->create([
+        'is_external' => true,
+    ]);
+
+    asSuperAdmin();
+
+    livewire(ViewUser::class, [
+        'record' => $user->getRouteKey(),
+    ])
+        ->assertActionHidden('mfa_status');
+});
+
+it('displays the proper mfa_status Action for an internal User without MFA enabled', function () {
+    $user = User::factory()->create([
+        'is_external' => false,
+    ]);
+
+    asSuperAdmin();
+
+    $component = livewire(ViewUser::class, [
+        'record' => $user->getRouteKey(),
+    ]);
+
+    $component
+        ->assertActionHasLabel('mfa_status', 'MFA Disabled')
+        ->assertActionHasColor('mfa_status', 'gray');
```

Comment:
You don't do anything with the component.

```suggestion
    livewire(ViewUser::class, [
        'record' => $user->getRouteKey(),
    ])
        ->assertActionHasLabel('mfa_status', 'MFA Disabled')
        ->assertActionHasColor('mfa_status', 'gray');
```